README.md

```diff
@@ -7,7 +7,7 @@
 
 This diagram represents how data gets loaded into the application:
 
-![overview diagram](https://docs.google.com/drawings/d/e/2PACX-1vTBFJJgiPqs58fNWC-lTBdw5wKNN0-OgLBu7EUoJcfyDXFu6VTKkhxNUKcNSX4f1Mf_mHHI2zH_ezZj/pub?w=960&h=720)
+![overview diagram](https://github.com/sul-dlss/dlme/blob/master/README.md)
```

Comment:
Is your intention here to remove the image from the README?

Reply:
I pasted the wrong url. Should be fixed now